packages/calypso-e2e/src/lib/pages/themes-detail-page.ts

```diff
@@ -61,6 +65,34 @@ export class ThemesDetailPage {
 		}
 	}
 
+	/**
+	 * Click on the Pick this design button displayed in Logged out theme details.
+	 */
+	async pickThisDesign(): Promise< void > {
+		await this.page.click( selectors.pickDesignButton );
+	}
+
+	/**
+	 * Visit the theme details page.
+	 *
+	 * @param themeSlug
+	 * @param siteSlug
+	 */
+	async visitTheme( themeSlug: string, siteSlug: string | null = null ) {
+		const targetUrl = `theme/${ themeSlug }/${ siteSlug ?? '' }`;
+
+		// We are getting a pending status for https://wordpress.com/cspreport intermittently
+		// which causes the login to hang on networkidle when running the tests locally.
+		// This fulfill's the route request with status 200.
+		// See https://github.com/Automattic/wp-calypso/issues/69294
```

Comment:
**question** - this is happening when the theme detail page is viewed while logged out, correct?

Reply:
Yes :)